### ISSUE TITLE
Functions for the Testing

### ISSUE THREAD

**Issue by NSchiciano** [NSchiciano/ClassWorkSpring2021#3]
https://github.com/NSchiciano/ClassWorkSpring2021/blob/4a83da014daa6606ed8d850ab055138de5792f3e/ClassExLine.py#L1

**Comment by dward2** [comment]
I saw all of your issues.  Thanks.

**Comment by NSchiciano** [comment]
Sorry if it was spam, for future class activities, should I just do an issue at the end?

Nick
________________________________

**Comment by dward2** [comment]
No, I didn’t consider them spam.  I was simply acknowledging your work.  You did exactly what I asked you to do.  Open issues when you need them and have a question.  Dr. Ward
